disco_aws_automation/disco_vpc.py

```diff
@@ -537,6 +538,8 @@ def _destroy_instances(self):
         """ Find all instances in vpc and terminate them """
         autoscale = DiscoAutoscale(environment_name=self.environment_name)
         autoscale.clean_groups(force=True)
+        elastigroup = DiscoElastigroup(environment_name=self.environment_name)
+        elastigroup.delete_groups()
```

Comment:
DiscoAutoscale `clean_groups` here is used as `delete_groups` with **force=True**, so we can call delete_groups instead of clean_groups.